### PR TITLE
Integrate Export / Import into HTTP api

### PR DESCRIPTION
**Pull request by lferran**
### Description
Integrates export and import logic into our reader and writer components so that the feature is available via HTTP api

### How was this PR tested?
Integration and unit tests

### CODE REVIEW

File: nucliadb/nucliadb/writer/run.py [hunk 1]
```diff
@@ -18,8 +18,8 @@
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
 
+from nucliadb.reader.app import get_application
```
Comment (vangheem) [inline nucliadb/nucliadb/writer/run.py:21]: you're calling get_application from reader in the writer component!

File: nucliadb/nucliadb/reader/app.py [hunk 2]
```diff
@@ -81,25 +83,29 @@ async def client_disconnect_handler(request: Request, exc: ClientDisconnect):
 )
 
 
-base_app = FastAPI(title="NucliaDB Reader API", **fastapi_settings)  # type: ignore
+def get_application() -> FastAPI:
```
Comment (vangheem) [inline nucliadb/nucliadb/reader/app.py:86]: I wonder is "create_application" would be a better word actually?

Comment (lferran) [reply nucliadb/nucliadb/reader/app.py:86]: yup. or application factory?

File: nucliadb/nucliadb/writer/run.py (hunk 1 above)
Comment (lferran) [reply nucliadb/nucliadb/writer/run.py:21]: haha no wonder why tests were not working

**Review by jotare — APPROVED** [review]

**Pull request merged**